components/collection/unlockable/UnlockableLandingTag.vue

```diff
@@ -41,7 +41,7 @@
     <div class="separator mx-2" />
     <nuxt-link
       class="is-flex is-align-items-center has-text-weight-bold my-2"
-      to="/ahp/drops/free-drop">
+      to="/ahp/drops/berlin-drop">
```

Comment:
we'll still keep this hardcoded url?

Reply:
Yes, we should keep it unless we redirect the user to the drops list page.